src/main/java/com/joss/conductor/mobile/Locomotive.java

```diff
@@ -71,7 +74,9 @@ public Locomotive(ConductorConfig configuration, AppiumDriver driver) {
 
     @Before
     @BeforeMethod(alwaysRun = true)
-    public void init() {
+    public void init(Method method) {
```

Comment:
Neat, didn't know TestNG did dependency injection, does it also work for JUnit?

Reply:
I think this is TestNG specific: http://testng.org/doc/documentation-main.html#native-dependency-injection

Reply:
Thanks for pointing this out though, my current implementation breaks running tests through jUnit. I will update this so the tests can be run at least through jUnit even if the test name won't be set correctly.